<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GalPatternDisplay</class>
 <widget class="QWidget" name="GalPatternDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pattern Display</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="5">
    <widget class="KPlotWidget" name="Plotwidget"/>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="UnzoomButton">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Unzoom pattern to maximum&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../galapicons.qrc">
       <normaloff>:/icons/zoomoff.png</normaloff>:/icons/zoomoff.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="SetZoomButton">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pick pattern range&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../galapicons.qrc">
       <normaloff>:/icons/zoomlim.png</normaloff>:/icons/zoomlim.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QSpinBox" name="PatternLow_spinBox">
     <property name="toolTip">
      <string>Lowel limit of selected pattern</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QSpinBox" name="PatternHi_spinBox">
     <property name="toolTip">
      <string>Upper imit of selected pattern</string>
     </property>
     <property name="maximum">
      <number>65536</number>
     </property>
     <property name="value">
      <number>1024</number>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>636</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPlotWidget</class>
   <extends>QFrame</extends>
   <header>kplotwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../galapicons.qrc"/>
 </resources>
 <connections/>
</ui>
